media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format.  flac123 is the only \
+component of flac-tools at this time."
+HOMEPAGE="http://flac-tools.sourceforge.net/"
+COPYRIGHT="2003-2017 Jake Angerman"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/flac-tools/flac123-$portVersion-release.tar.gz"
```

Comment:
Well, not everyone can use https unfortunately. We had at least one GCI student this year whose ISP has a "transparent" proxy for HTTPS, serving its own certificate so it can spy on HTTPS connections. This is a problem because Haiku does not recognize the certificate (for good reasons) and there is no easy way to add it.
However, we should support the use of SSL secure connections whenever possible.